javaScript/index.js

```diff
@@ -41,4 +41,120 @@ function handleScreenChange(event) {
 
 handleScreenChange(mobileQuery);
 
-mobileQuery.addEventListener('change', handleScreenChange);
\ No newline at end of file
+mobileQuery.addEventListener('change', handleScreenChange);
+
+const projects = [
+  {
+    cardName: 'Multi-Post Stories Gain+Glory',
+    technologies: `
+    <li>Ruby on Rails</li>
+    <li>CSS</li>
+    <li>JavaScript</li>
+    <li>HTML</li>`,
+    cardSource: '<p class="btn"><a href="#" id="popupbtn1">See project</a></p>',
+  },
+  {
+    cardName: 'Multi-Post Stories Gain+Glory',
+    technologies: `
+    <li>Ruby on Rails</li>
+    <li>CSS</li>
+    <li>JavaScript</li>
+    <li>HTML</li>`,
+    cardSource: '<p class="btn"><a href="#" id="popupbtn2">See project</a></p>',
+  },
+  {
+    cardName: 'Multi-Post Stories Gain+Glory',
+    technologies: `
+    <li>Ruby on Rails</li>
+    <li>CSS</li>
+    <li>JavaScript</li>
+    <li>HTML</li>`,
+    cardSource: '<p class="btn"><a href="#" id="popupbtn3">See project</a></p>',
+  },
+  {
+    cardName: 'Multi-Post Stories Gain+Glory',
+    technologies: `
+    <li>Ruby on Rails</li>
+    <li>CSS</li>
+    <li>JavaScript</li>
+    <li>HTML</li>`,
+    cardSource: '<p class="btn"><a href="#" id="popupbtn4">See project</a></p>',
+  },
+  {
+    cardName: 'Multi-Post Stories Gain+Glory',
+    technologies: `
+    <li>Ruby on Rails</li>
+    <li>CSS</li>
+    <li>JavaScript</li>
+    <li>HTML</li>`,
+    cardSource: '<p class="btn"><a href="#" id="popupbtn5">See project</a></p>',
+  },
+  {
+    cardName: 'Multi-Post Stories Gain+Glory',
+    technologies: `
+    <li>Ruby on Rails</li>
+    <li>CSS</li>
+    <li>JavaScript</li>
+    <li>HTML</li>`,
+    cardSource: '<p class="btn"><a href="#" id="popupbtn6">See project</a></p>',
+  },
+];
+
+const workSection = document.getElementById('worksection');
+
+for (let i = 0; i < projects.length; i += 1) {
+  const fetch = workSection.innerHTML;
+  workSection.innerHTML = `
+  <div class="card item1">
+    <div class="card-info">
+      <h3>${projects[i].cardName}</h3>
+      <ul>
+        ${projects[i].technologies}
+      </ul>
+      <div class="btnbar">
+        ${projects[i].cardSource}
+      </div>
+    </div>
+  </div> ${fetch}`;
+}
+
+const popupOn1 = document.getElementById('popupbtn1');
+const popupOn2 = document.getElementById('popupbtn2');
+const popupOn3 = document.getElementById('popupbtn3');
+const popupOn4 = document.getElementById('popupbtn4');
+const popupOn5 = document.getElementById('popupbtn5');
+const popupOn6 = document.getElementById('popupbtn6');
+const popupOff = document.getElementById('popup-offbtn');
+const modalOff = document.getElementById('popup');
+const popUp = document.getElementById('popup-wrapper');
+
+function openModal() {
+  popUp.style.display = 'block';
+}
```

Comment:
- [ ] Kindly make sure for each button clicked, the correct data are populated on the pop_up modal, I mean the `title`, `description`, `...` for the specific clicked project, All the pop-up modals are not supposed to display the same contents as you have done.

Reply:
Can you please clarify these points? Firstly you approved my project but you required changes I don't understand, why?
And I don't know have you noticed or not but the template I choose have the same type of 6 cards with the same information or description. That is why the modals are the same too.

Reply:
Hey @Al-Amin-Khan-Shakil , Sorry for the mistake of the other time, Instead of requesting changes I have written Approved
I mean you can put the same card as placeholders bu the project description, title will not be the same, If you understand what I mean

Reply:
Yes, I understand you and made the changes you requested and merge that pull request. But you changed the status again now what should I do? I have no pull request to submit a review request.